tests/lib/parse.js

```diff
@@ -105,5 +105,19 @@ describe("parse()", () => {
             );
         });
 
+        // https://github.com/eslint/espree/issues/470
+        it("Should throw on invaid `(a = 1) = t`", () => {
+            assert.throws(() => {
+                espree.parse("(a = 1) = t");
+            });
+        });
+
+        // https://github.com/eslint/espree/issues/472
+        it("Should throw on invaid `async () => await 5 ** 6;`", () => {
+            assert.throws(() => {
+                espree.parse("async () => await 5 ** 6;");
+            });
```

Comment:
Similar to the above, we should set a higher `ecmaVersion` to confirm that https://github.com/eslint/espree/issues/472 has been fixed, because this code with the default `ecmaVersion` already has syntax errors aside from `await 5 ** 6`.

Actually, it seems that this particular example doesn't throw in the latest Acorn 8.

Reply:
interesting! if adding a `{}`, it will throw an error.

```js
> acorn.parse('async() => { await 5 ** 6 }', {ecmaVersion: 10})
Uncaught [SyntaxError: Unexpected token (1:21)
] {
  pos: 21,
  loc: Position { line: 1, column: 21 },
  raisedAt: 23
}
```
sounds like a bug in acorn?

Reply:
just removed the case, as it was not fixed in latest acorn. :)